pages/newsletter/index.tsx

```diff
@@ -0,0 +1,164 @@
+import whiteLogo from "./assets/white_logo@1x.png";
+import { NewsletterEmailSubscribe } from "components/Newsletter/NewsletterEmailSubscribe";
+
+const Newsletter = () => {
+  return (
+    <>
+      <Head title="Newsletter" description="Subsribe to our Newsletter!" />
+      <FlexBG
+        backgroundImage={[`url(${mobBackground})`, `url(${webBackground})`]}
+        minHeight="100vh"
+      >
+        <Box m={["62px 0px 32px", "48px 0px 32px"]}>
+          <Box as="h1" m={"0 0 0 0"}>
+            <a href="/">
+              <Flex
+                justifyContent="center"
+                alignItems="top"
+                height={["24px", null]}
+                width={["120px", null]}
+              >
+                <Image
+                  src={whiteLogo}
+                  alt="network infrastructure"
+                  width={"100%"}
+                />
+              </Flex>
+            </a>
+          </Box>
+        </Box>
+        <Card>
+          <AccessPlane order={[2, 1]}>
+            <FlexBG
+              backgroundImage={`url(${engineers})`}
+              width={[282, 534]}
+              height={[158, 300]}
+              backgroundRepeat="no-repeat"
+              ml={[5, -3]}
+            />
+            <Box>
+              <Box
+                fontSize={["header-4", "header-3"]}
+                lineHeight={["22px", "24px"]}
+                fontWeight={"bold"}
+                width="100%"
+                p={["6px 16px 10px 18px", "12px 56px 14px 56px"]}
+              >
+                Access Plane
+              </Box>
+              <Box
+                fontSize={["15px", "text-lg"]}
+                lineHeight={["20px", "24px"]}
+                fontWeight={"regular"}
+                width="100%"
+                px={["18px", 8]}
+              >
+                Teleport allows engineers and security professionals to unify
+                access for SSH servers, Kubernetes clusters, web applications,
+                and databases across all environments.
+              </Box>
+            </Box>
+          </AccessPlane>
+          <EmailCTA>
+            <Flex
+              flexDirection="column"
+              justifyContent="space-betweem"
+              alignItems="center"
+              width={[343, 440]}
+              height={[298, 392]}
+            >
+              <Box
+                width={["307px", "304px"]}
+                display="flex"
+                flexDirection="column"
+                justifyContent="center"
+                alignItems="center"
+              >
+                <Box
+                  fontSize={["22px", "34px"]}
+                  lineHeight={["28px", "40px"]}
+                  width="100%"
+                  pt={[7, "100px"]}
+                  pb={[2, 3]}
+                  fontWeight={"bold"}
+                >
+                  Subscribe to our newsletter!
+                </Box>
+                <Box fontSize={["17px", "18px"]} width="100%" pb={[null, 3]}>
```

Comment:
```suggestion
                <Box fontSize="text-xl" width="100%" pb={[0, 3]}>
```

`17px` is not in our theme and too close to `18px` anyway. If it not fit mobile version, reduce it to `16px`.